ttax/base_class.py

```diff
@@ -24,6 +24,10 @@ def axis_dim(self):
   def batch_shape(self):
     return self.tt_cores[0].shape[:self.num_batch_dims]
 
+  @property
+  def tt_ranks(self):
```

Comment:
When adding properties to base class, unfortunately you need to also add them to the WrappedTT class https://github.com/fasghq/ttax/blob/develop/ttax/compile.py#L33